contracts/modules/RegistryController.sol

```diff
@@ -188,9 +188,19 @@ contract RegistryController is
 
         require(
             EnumerableSet.length(_contractNames[_release]) < MAX_CONTRACTS,
-            "ERROR:REC-005:MAX_CONTRACTS_LIMIT"
+            "ERROR:REC-010:MAX_CONTRACTS_LIMIT"
         );
 
+        require(_contractsInRelease[_release] > 0, "ERROR:REC-011:RELEASE_UNKNOWN");
```

Comment:
isn't a release initially empty?

Reply:
oh yes it is. :-( Well then, we need some other way to validate the `_release` attribute.

Reply:
@matthiaszimmermann i pushed a workaround for the `prepareRelease` case. I don't like it, but i don't have a better idea right now. Can you please have a look, maybe you'll find something better.